lib/slim_fast/parser.ex

```diff
@@ -95,6 +71,40 @@ defmodule SlimFast.Parser do
     end
   end
 
+  @attr_regex ~r/([\w-]+)\s*=\s*(".+"|[\w-]+)/
```

Comment:
Probably should include single quotes too.  Something like this perhaps: `\s*(["'].+['"]|#{.+}|[@\w.-]+)`